app/uk/gov/hmrc/eusubsidycompliancefrontend/util/EmailTemplateHelpers.scala

```diff
@@ -43,7 +45,7 @@ object TemplateHelpers {
       case other        => sys.error(s"Found unsupported language code $other")
     }
 
-  def getTemplateId(configuration: Configuration, inputKey: String
+  def getEmailTemplateId(configuration: Configuration, inputKey: String
```

Comment:
This is fine as it is, but we could look at either providing our own configuration class with additional methods like this, or perhaps offering configuration syntax helpers (like the FutureSyntax or TaxYearSyntax classes). I don't think this needs to be changed now though.

Reply:
Sorry i don't get what are you trying to say here. I am doing this in a separate helper class EmailTemplateHelpers.

Reply:
Messaged you on slack. Nothing needs to be changed here. This was just a general comment.